documentation/running_bolt_commands.md

```diff
@@ -128,6 +128,56 @@ best limited to small files.
     ```
     bolt file upload my_file.txt /tmp/remote_file.txt --targets web5.mydomain.edu,web6.mydomain.edu
     ```
+
+## Download files or directories from remote targets
+
+Use Bolt to copy files or directories from remote targets to your local system.
+
+To download a file or directory from a remote target, run the
+`bolt file download` command. Specify the remote path to the file or
+directory, the destination directory on your local system, and the targets.
+
+```shell
+$ bolt file download <SOURCE> <DESTINATION> --targets <TARGETS>
+```
+
+The `destination` can be either an absolute or relative path to a directory
+on your local system. If a relative path is given, the path is expanded
```

Comment:
Just double-checking something: `downloads` is only used when you use a relative path? So if I gave an absolute path, I could specify any directory name and the result would not be nested under a `downloads` directory?

Reply:
Correct

Reply:
Okay cool, I think this bit is good then 👍